### ISSUE TITLE
Consider adding classifications to the lint list

### ISSUE THREAD

**Issue by rmhrisk** [zmap/zlint#228]
Right now each lint has a severity and a citation, it would be useful if the lints also had a classification, for example:
- Extended Validation Errors
- Validation Errors
- Encoding Errors

This would allow the tools that utilize zlint to quickly filter through the issues to get to the most pressing things within that classification.

**Comment by sdavidson-quovadis** [comment]
I agree this would be useful - as issues in these buckets typically differ in resulting from a "process problem" and "technology problem".
